src/speed.h

```diff
@@ -0,0 +1,32 @@
+#ifndef __SPEED_H
+#define __SPEED_H
+
+#include "setup.h"
+
+/* Speed modes*/
+#define SLOW_MODE 0
+#define NORMAL_MODE 1
+#define FAST_MODE 2
```

Comment:
I'd say define this as an `enum` or do not define those speed names at all.

Reply:
I will not define them at the moment